### PR TITLE
Fix - ui-radio checked property is triggering when value changes.

### PR DESCRIPTION
**Pull request by gowtham95**

### CODE REVIEW

**Comment by aaronbhansen** [comment]
I don't see this as an issue. If value or current changes, you need checked to recompute to see if value is now equal to current or not. The current code is master is correct. Why would you not want it to recompute when the values change?

**Comment by gowtham95** [comment]
@aaronbhansen  In documentation you have given an example 
`{{ui-radio name="fruit" label="Once a week" value="apple" current=fruit}}` where you have passed `current`, but you are actually passing `checked=fruit`. try passing current=fruit.

**Comment by gowtham95** [comment]
Checking for exact issue will update you once I get it.

**Comment by gowtham95** [comment]
In semantic-ui you are setting `checked` in `module.set.checked()` right ? then why  need  to re-compute when `current` changes?

**Comment by aaronbhansen** [comment]
Checked is really there as a helper, incase you wanted to extend the base class, you would have a property to let you know if its checked or not. It's not really used or to be set. Also on the dummy app, that hasn't been updated in a while and probably should be removed.

**Comment by aaronbhansen** [comment]
I think the actual issue is related to this bug https://github.com/Semantic-Org/Semantic-UI-Ember/pull/82 If you want to make the changes based on the comments we can get the test changes merged in at least

File: addon/components/ui-radio.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default Ember.Component.extend(CheckboxMixin, {
     this.set('current', this.get('value'));
   },
 
-  checked: Ember.computed('current', 'value', function() {
+  checked: Ember.computed(function() {
```
Comment (aaronbhansen) [inline addon/components/ui-radio.js]: Lets revert this change

File: tests/dummy/app/templates/modules/checkbox.hbs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <h2 id='title'>Checkbox</h2>
 
-{{ui-checkbox label="Make my profile visible (checked binding)" checked=checked}}
+{{!--ui-checkbox label="Make my profile visible (checked binding)" checked=checked}}
```
Comment (aaronbhansen) [inline tests/dummy/app/templates/modules/checkbox.hbs]: Why did you comment out these?

**Comment by gowtham95** [comment]
Yeah this is related to this bu #82 hence closing this pull request.
